eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -0,0 +1,31 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $serviceName,
+  [Parameter(Mandatory=$True)]
+  [string] $OutDirectory
+)
+
+. (Join-Path $PSScriptRoot common.ps1)
+$allPackageProperties = Get-AllPkgProperties $serviceName
+if ($allPackageProperties)
+{
+    New-Item -ItemType Directory -Force -Path $OutDirectory
+    foreach($pkg in $allPackageProperties)
+    {
+        Write-Host "Package Name: $($pkg.Name)"
+        Write-Host "Package Version: $($pkg.Version)"
+        Write-Host "Package SDK Type: $($pkg.SdkType)"
+        $outputPath = Join-Path -Path $OutDirectory ($pkg.Name + ".json")
```

Comment:
I believe Name might not be enough for things like Java. Where there are duplicate names and sometimes we might need to add in group id to disambiguate.

Reply:
As discussed this step will exclude dumping properties for track 1 packages